### PR TITLE
Initial stylings

### PR DESCRIPTION
**Pull request by DariaSova**

### CODE REVIEW

File: app/assets/stylesheets/layouts/_main-content.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.main-content {
+  position: relative;
+  top: 0px;
```
Comment (Sinetheta) [inline app/assets/stylesheets/layouts/_main-content.scss]: I don't think this is doing anything.

File: app/assets/stylesheets/layouts/_main-content.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  left: 25%;
```
Comment (Sinetheta) [inline app/assets/stylesheets/layouts/_main-content.scss]: This now depends on the width. If you want to centre this container you might just want `auto` on the left and right.

File: app/assets/stylesheets/layouts/_main-content.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  width: 50%;
+  margin-bottom: 12%;
```
Comment (Sinetheta) [inline app/assets/stylesheets/layouts/_main-content.scss]: Is there anything that appears underneath the `.main-content` to push down with this?

File: app/assets/stylesheets/components/_buttons.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+button,
+input[type="button"],
+input[type="reset"],
+input[type="submit"],
+.button-link {
+  margin-top: 15px;
```
Comment (Sinetheta) [inline app/assets/stylesheets/components/_buttons.scss]: All buttons have a top margin? Maybe a style change would be made somewhere else to do what this spacing is doing.

File: app/assets/stylesheets/_typography.scss [hunk 5]
```diff
@@ -0,0 +1,5 @@
+.main-title {
+  color: $color-title-main;
+  font-size: 300%;
```
Comment (Sinetheta) [inline app/assets/stylesheets/_typography.scss]: Usually `em` are used in place of `%` for font-size.

File: app/assets/stylesheets/_color_variables.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+$color-title-main: white;
+$color-body: #E3DEC1;
+$color-buttons-main: #eee;
```
Comment (Sinetheta) [inline app/assets/stylesheets/_color_variables.scss:3]: You might want to choose one style of describing colors to use throughout a project.

File: app/assets/stylesheets/_typography.scss (hunk 5 above)
Comment (DariaSova) [inline app/assets/stylesheets/_typography.scss]: is there a big difference or a reason for that?

File: app/assets/stylesheets/layouts/_main-content.scss (hunk 3 above)
Comment (DariaSova) [inline app/assets/stylesheets/layouts/_main-content.scss]: nope.. so I guess I don't need margin-bottom at all

File: app/assets/stylesheets/layouts/_main-content.scss (hunk 2 above)
Comment (DariaSova) [inline app/assets/stylesheets/layouts/_main-content.scss]: :+1:

File: app/assets/stylesheets/_color_variables.scss (hunk 6 above)
Comment (DariaSova) [inline app/assets/stylesheets/_color_variables.scss:3]: is it ok if I do it later in the project? at this point I chose pretty much random colours just to setup the variables

**Comment by Sinetheta** [comment]
:+1:

**Pull request merged**